anchore_engine/clients/skopeo_wrapper.py

```diff
@@ -331,6 +331,135 @@ def get_image_manifest_skopeo(url, registry, repo, intag=None, indigest=None, to
+        else:
+            tlsverifystr = "--tls-verify=false"
+
+        localconfig = anchore_engine.configuration.localconfig.get_config()
+        global_timeout = localconfig.get('skopeo_global_timeout', 0)
+        try:
+            global_timeout = int(global_timeout)
+            if global_timeout < 0:
+                global_timeout = 0
+        except:
+            global_timeout = 0
+
+        if global_timeout:
+            global_timeout_str = "--command-timeout {}s".format(global_timeout)
+        else:
+            global_timeout_str = ""
+
+        os_override_strs = ["", "--override-os windows"]
+        try:
+            success = False
+            for os_override_str in os_override_strs:
+                cmd = ["/bin/sh", "-c",
+                       "skopeo {} {} inspect {} {} docker://{}".format(global_timeout_str, os_override_str,
+                                                                       tlsverifystr, credstr, pullstring)]
+                cmdstr = ' '.join(cmd)
+                try:
+                    rc, sout, serr = run_command_list(cmd, env=proc_env)
+                    if rc != 0:
+                        skopeo_error = SkopeoError(cmd=cmd, rc=rc, out=sout, err=serr)
+                        if skopeo_error.error_code != AnchoreError.OSARCH_MISMATCH.name:
+                            raise SkopeoError(cmd=cmd, rc=rc, out=sout, err=serr)
+                    else:
+                        logger.debug(
+                            "command succeeded: cmd=" + str(cmdstr) + " stdout=" + str(sout).strip() + " stderr=" + str(
+                                serr).strip())
+                        success = True
+                except Exception as err:
+                    logger.error("command failed with exception - " + str(err))
+                    raise err
+
+                if success:
+                    sout = str(sout, 'utf-8') if sout else None
+                    ret = sout
+                    break
+
+            if not success:
+                logger.error("could not retrieve manifest")
+                raise Exception("could not retrieve manifest")
+
+        except Exception as err:
+            raise err
+    except Exception as err:
+        raise err
+
+    return ret
+
+def _get_image_manifest_digest(registry, repo, tag=None, digest=None, user=None, pw=None, verify=True):
+    if digest:
+        pull_string = registry + "/" + repo + "@" + digest
+    elif tag:
+        pull_string = registry + "/" + repo + ":" + tag
+    else:
+        raise Exception("invalid input - must supply either an intag or indigest")
+
+    try:
+        raw_manifest = get_image_manifest_skopeo_raw(pull_string, user=user, pw=pw, verify=verify)
+        s_manifest = json.loads(raw_manifest)
+        s_digest = manifest_to_digest(raw_manifest)
+        return s_manifest, s_digest
+    except Exception as err:
+        logger.warn("CMD failed - exception: " + str(err))
```

Comment:
this doesn't add anything that the Exception wouldn't already show. Does this imply there is something beyond `str(err)` that is useful to log? It seems this could do well without a `try/except` block